rebar.config

```diff
@@ -22,8 +22,8 @@
 {profiles,
     [{test,
         [{deps,
-            [{meck, "0.8.13"},
-             {emqx, {git, "https://github.com/emqx/emqx", {branch, "master"}}},
+            [{meck, "0.9.2"},
+             {emqx, {git_subdir, "https://github.com/emqx/emqx", {branch, "master"}, "apps/emqx"}},
```

Comment:
awesome

Reply:
so this is the emqx's rebar3 feature?

Reply:
@qzhuyan it's an official feature http://rebar3.org/docs/configuration/dependencies/